src/questions/Questions.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const Questions = (props) =>{
+    function selectQuestion(e){
+        e.currentTarget.classList.toggle("selected")
+      }
+    const faq = props.faq;
+    return(
+        <div className="questions">
+            <h1 className="logo">FAQ</h1>
+            <ul className="question-list">
+                {faq.map((curr)=>{
```

Comment:
Use meaningful variable names, e.g `faqItem`, `faqQuestion`. As well as for faq, `faqItems`, `faqList`